api/config/esgateway/config_request.go

```diff
@@ -48,8 +48,9 @@ func DefaultConfigRequest() *ConfigRequest {
 	// resolvers once we are a bit more confident in the nginx
 	// module we are using to do dynamic resolution.
 	//
-	// c.V1.Sys.Ngx.Main.Resolvers = getSystemResolvers()
 
+	c.V1.Sys.Ngx.Main.Resolvers.EnableSystemNameservers = w.Bool(false)
```

Comment:
`v.V1.Sys.Ngx.Main.Resolvers` is going to be nil. You'll want to initialize it to an empty struct in `NewConfigRequest`